sql/updates/latest-dev.sql

```diff
@@ -10,3 +10,14 @@ CREATE TABLE _timescaledb_catalog.dimension_partition (
 SELECT pg_catalog.pg_extension_config_dump('_timescaledb_catalog.dimension_partition', '');
 GRANT SELECT ON _timescaledb_catalog.dimension_partition TO PUBLIC;
 DROP FUNCTION IF EXISTS @extschema@.remove_continuous_aggregate_policy(REGCLASS, BOOL);
+
+-- add a new column to chunk catalog table 
+ALTER TABLE _timescaledb_catalog.chunk ADD COLUMN  osm_chunk boolean ;
+UPDATE _timescaledb_catalog.chunk SET osm_chunk = FALSE;
+
+ALTER TABLE _timescaledb_catalog.chunk
+  ALTER COLUMN  osm_chunk SET NOT NULL;
+ALTER TABLE _timescaledb_catalog.chunk 
+  ALTER COLUMN  osm_chunk SET DEFAULT FALSE;
+
+CREATE INDEX chunk_osm_chunk_idx ON _timescaledb_catalog.chunk (hypertable_id, osm_chunk);
```

Comment:
I'd suggest `(osm_chunk, hypertable_id)` to keep the relevant part of the index sequential. Also let's add a comment about why we are not using a partial index.

Also we could use `using chunk_id` and an index-only scan for this query, as I described in the previous PR, but this is somewhat optional.

Reply:
And also maybe a comment about why we are not using the status field -- can't use index for bit operations and can't be sure that the frozen status is not combined with the other ones, so can't use the equality lookup.

Reply:
modified index order and added a comment.